app.js

```diff
@@ -1,55 +1,49 @@
-const computerChoiceDisplay = document.getElementById("computer-choice")
-const userChoiceDisplay = document.getElementById("user-choice")
-const resultDisplay = document.getElementById("result")
-const possibleChoices = document.querySelectorAll("button")
-let userChoice
-let computerChoice
-let result
+const resultMap = {
+    'rock': {
+        'paper': false,
+        'scissors': true,
+    },
+    'paper': {
+        'rock': true,
+        'scissors': false,
+    },
+    'scissors': {
+        'rock': false,
+        'paper': true
+    }
+};
```

Comment:
Working against a dictionary / object cuts down on the repetition - there's probably still a nicer way than this, but it's considerably more elegant and extensible than a list of `if`s